### DISCUSSION TITLE
what is WIP and TCO?

### DISCUSSION THREAD

**Discussion by yanqingfu** [Seagate/cortx#455] [Q&A] [answered]
https://github.com/Seagate/cortx/blob/main/FAQs.md

> 'This is a WIP FAQ'

> 'the TCO benefits that come with that approach'

**Comment by justinzw** [comment]
Thanks @yanqingfu  WIP= Work in Progress.

I'm going to to make a pull request to remove that as I don't think it's needed.

@johnbent is TCO = Total Cost of Ownership?

**Comment by johnbent** [comment, marked as answer]
Yes, about TCO.  We merged the PR.  Thanks both!
